accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+
+# Definitions
+## SDK Band
+
+.NET SDK versions take the form 3.1.100, 3.1.101, 3.1.200. The first two components of this version represent the major and minor version of the runtime in the SDK. The third component (the “hundreds”) is a more complex: the first digit represents the feature release of the SDK, and the last two digits represent the patch level of the SDK and runtime combined.
+
+In the context of this document we will refer to an *SDK band* as the common portion of the version of all the SDK releases that differ only by patch level. For example, 3.1.100 and 3.1.105 are in the 3.1.100 band, while 3.2.100 is in the 3.2.100 band and 3.1.203 is in the 3.1.200 band.
+
+## SDK Pack
+
+An SDK pack is a unit of packaging for the assets that make up the SDK. Packs have an ID and version and are installed into folders in the SDK directory.
+
+Packs will be installed using CLI tooling or by a native installer such as the Visual Studio installer. It is likely that packs will be distributed via NuGet at some point, so it is recommended that pack authors reserve NuGet IDs for their packs even if other distribution methods are used initially.
+
+SDK packs are used for construction, composition and distribution only, they are not exposed through the user experience.
+
+## SDK Workload
+
+An SDK workload is a semantic name describing a set of one or more SDK packs. When a developer wants to be able to build a particular kind of app, they would expect to install a workload. Workloads should correspond to user scenarios.
+
+Here are some examples of SDK workloads and the packs that they might include:
+
+| Workload ID | Content|
+|--|--|
+| xamarin-android | Framework, runtime, build logic, tools and templates for Xamarin.Android.|
+| xamarin-forms | Templates for Xamarin.Forms and the Xamarin.Forms NuGet package used by the templates.<br><br>This would also include any packs necessary to build and run Xamarin.Forms projects. That could be done by depending on the xamarin-ios, xamarin-android and uwp components, or perhaps instead some kind of lightweight Xamarin.Forms host. |
+| xamarin-forms-android | Templates for Xamarin.Forms and the Xamarin.Forms NuGet package used by the templates.<br><br>This would also include any packs necessary to build and run Xamarin.Forms projects on Android by depending on xamarin-android.|
+| xamarin-android-llvm | The LLVM AOT compiler for Android, omitted by default from the xamarin-android component because it’s large and not all apps use it.<br><br>It may make sense for this to depend on the xamarin-android workload as it’s useless by itself.|
+
+# Versioning
+
+## Manifest Versioning
+
+Manifests follows a similar pattern to SDK versioning: each SDK band has a corresponding manifest band. Each SDK uses only the manifest band corresponding to its own band. Manifest versions however are completely independent of SDK patch levels; an SDK should use the latest available versions of the manifests for its band.
```

Comment:
> each SDK band has a corresponding manifest band

The introduction of a manifest band adds too much cognitive load. 

> an SDK should use the latest available versions of the manifests for its band

This is clear. My take is that the SDK always uses the latest manifest for sdk band. It's possible there will be an advanced scenario to do something different (would be quite useful for testing).

Reply:
Rewrote to avoid introducing the term "manifest band"